site/gatsby-site/src/templates/post.js

```diff
@@ -51,7 +51,7 @@ export default function Post(props) {
           page="post"
         ></SocialShareButtons>
       </div>
-      <StyledMainWrapper>
+      <StyledMainWrapper className="max-w-full">
```

Comment:
I think full-width is good for the blog index, but for the posts the readability issue of long lines still applies. I would just revert the change on this line, and then it will be good to go.

Reply:
@lmcnulty done!